apps/api/src/app/integrations/usecases/get-integration/get-integration.usecase.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+import { IntegrationEntity, IntegrationRepository } from '@notifire/dal';
+import { GetIntegrationCommand } from './get-integration.command';
+
+@Injectable()
+export class GetIntegration {
```

Comment:
```suggestion
export class GetIntegrations {
```

I think this should be fetch integrations, correct?